autoscheduler/frontend/src/tests/ReduxAvailability.test.tsx

```diff
@@ -155,6 +155,61 @@ describe('Availabilities', () => {
       // assert
       expect(store.getState().availability).toEqual(expected);
     });
+
+    // doesn't seeem to be doing anything
```

Comment:
Why is this comment here? Does this test not properly fail with the old code (since the problem seems inconsistent to replicate) or something else

Reply:
old comment, should've been removed when i fixed it